app/helpers/application_helper/toolbar/openstack_vm_cloud_center.rb

```diff
@@ -69,6 +69,22 @@ class ApplicationHelper::Toolbar::OpenstackVmCloudCenter < ApplicationHelper::To
           t,
           :klass => ApplicationHelper::Button::InstanceDisassociateFloatingIp),
         button(
+          :instance_add_security_group,
+          'pficon pficon-cloud-security fa-lg',
+          t = N_('Add a Security Group to this Instance'),
+          t,
+          :klass   => ApplicationHelper::Button::GenericFeatureButtonWithDisable,
+          :options => {:feature => :add_security_group}
```

Comment:
This is probably not needed..  `instance_add_security_group` is the needed rbac feature, because of the toolbar button's identifier.

Reply:
Hi! I had thought that the preference in feature names was to use non-entity specific names when possible. It's certainly possible that we'll be adding security groups to other things in the future, so isn't add_security_group preferable?

Reply:
Well until that happens I'd go with the simple case.

Other that that I think this is ready to merge.

Reply:
I understand the concern, but I've been explicitly told to do the opposite by reviewers who reviewed changes to the feature list in the supports_feature_mixin. If you look at https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb, none of the features are entity specific.

Reply:
Oh well @himdel and I both mixed up RBAC feature and product feature.